packages/hint-font-relative-units/src/hint.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * @fileoverview Ensure font styles use relative units to enable user stylesheet overrides
+ */
+
+import { Declaration, Rule } from 'postcss';
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+// The list of types depends on the events you want to capture.
+import { IHint, ProblemLocation } from 'hint/dist/src/lib/types';
+import { debug as d } from '@hint/utils';
+import { getCSSCodeSnippet } from '@hint/utils/dist/src/report/get-css-code-snippet';
+import { StyleEvents, StyleParse } from '@hint/parser-css';
+
+import meta from './meta';
+import { getMessage } from './i18n.import';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class FontA11yFontSizeUnitsHint implements IHint {
+
+    public static readonly meta = meta;
+    public readonly FONT_PROPERTIES = ['font-size', 'line-height', 'letter-spacing'];
+
+    public constructor(context: HintContext<StyleEvents>) {
+
+        const getLocation = (decl: Declaration): ProblemLocation => {
```

Comment:
This should be removed in favor of `getLocationFromNode` once https://github.com/webhintio/hint/pull/3138 is merged.